test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/MiddlewareAnalysisSampleTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Yeah we want to make sure the samples actually run and seem to approximately do whatever they intend to do.